library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionViewModel.kt

```diff
@@ -1,28 +1,54 @@
 package com.chuckerteam.chucker.internal.ui.transaction
 
 import androidx.lifecycle.LiveData
-import androidx.lifecycle.Transformations
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.map
 import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
 import com.chuckerteam.chucker.internal.data.repository.RepositoryProvider
+import com.chuckerteam.chucker.internal.support.combineLatest
 
-internal class TransactionViewModel(transactionId: Long) : ViewModel() {
+internal class TransactionViewModel(
+    transactionId: Long
+) : ViewModel() {
 
-    val transactionTitle: LiveData<String> =
-        Transformations.map(RepositoryProvider.transaction().getTransaction(transactionId)) {
-            if (it != null) "${it.method} ${it.path}" else ""
+    private val mutableEncodeUrl = MutableLiveData<Boolean>(false)
+
+    val encodeUrl: LiveData<Boolean> = mutableEncodeUrl
+
+    val transactionTitle: LiveData<String> = RepositoryProvider.transaction()
+        .getTransaction(transactionId)
+        .combineLatest(encodeUrl) { transaction, encodeUrl ->
+            if (transaction != null) "${transaction.method} ${transaction.getFormattedPath(encode = encodeUrl)}" else ""
```

Comment:
This line seems too long.
```suggestion
            if (transaction != null) {
                "${transaction.method} ${transaction.getFormattedPath(encode = encodeUrl)}"
            } else {
                ""
            }
```

Reply:
It looks long to me as well but it does not break Detekt rules. I can change it but I would rather rely on static analysis for such stuff. Otherwise what's the point of it?

Reply:
Ok, as you wish. I will update rules for static analysis soon.